README.md

```diff
@@ -65,8 +65,11 @@ When a commit is pushed into `X.Y`:
   * Notify the Doc team on the [TheForeman Doc chat](https://matrix.to/#/#theforeman-doc:matrix.org) Matrix channel
 * Update master
   * Update `ProjectVersionPrevious` to `X.Y` in `guides/common/attributes.adoc`
-  * Create a copy of [/web/content/release-nightly.md](https://github.com/theforeman/foreman-documentation/tree/master/web/content/release-nightly.md) as `release-X.Y.md` and edit it accordingly.
-  * Add the new release versions to the site's index page and navigation bar by editing [/web/content/index.adoc](https://github.com/theforeman/foreman-documentation/blob/master/web/content/index.adoc) and [/web/content/js/versions.js](https://github.com/theforeman/foreman-documentation/blob/master/web/content/js/versions.js).
+  * Create a copy of [/web/releases/nightly.json](https://github.com/theforeman/foreman-documentation/tree/master/web/releases/nightly.json) as `X.Y.json` and edit it accordingly. Remove guides which aren't ready for stable branches.
+  * Create a copy of [/web/releases/nightly.adoc](https://github.com/theforeman/foreman-documentation/tree/master/web/releases/nightly.adoc) as `X.Y.adoc` and edit it accordingly.
+    * Set the `state` to `RC`
+    * Add `katello` with the right version
+    * Remove guides which aren't ready for stable branches.
```

Comment:
I think .adoc needs other changes than listed here?

Reply:
Right, these apply to the `.json`.